### PR TITLE
673: added skip authorization on option sets child nodes

### PR DESCRIPTION
**Pull request by viniciuscb**
This commit solves the bug listed in #673

### CODE REVIEW

File: app/controllers/option_sets_controller.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ class OptionSetsController < ApplicationController
   # authorization via cancan
   load_and_authorize_resource
   skip_authorization_check only: :child_nodes
+  skip_authorize_resource only: :child_nodes
```
Comment (smoyth) [inline app/controllers/option_sets_controller.rb]: Why are both of these needed?

Comment (viniciuscb) [reply app/controllers/option_sets_controller.rb]: This is not related to our change in linked option sets, and is a bugfix in your code. The description of the bug is in the issue.

@smoyth I think I've not tested putting out skip_authorization_check.

**Comment by cooperka** [comment]
(Note: PRs #674, #672, #670, #668 were automatically closed due to removal of old `develop` branch; I've updated their bases and re-opened them)

**Comment by smoyth** [comment]
Closing for now due to age. Can be opened again later.

**Pull request closed without merging**